src/device-registry/models/Location.js

```diff
@@ -129,24 +128,25 @@ locationSchema.statics = {
       let createdAirQloud = await this.create({
         ...body,
       });
-      let data = jsonify(createdAirQloud);
-      if (!isEmpty(data)) {
+
+      if (!isEmpty(createdAirQloud)) {
+        let data = createdAirQloud._doc;
```

Comment:
resolving runtime error for cases where there is no records from the search filter